templates/contribute/index.md

```diff
@@ -108,6 +108,10 @@ Click on the "labels" header to add or remove labels from the current project.
 - `-awaiting-author` will remove the **"awaiting-author"** label
```

Comment:
To clarify the sentence above: anybody can edit the labels of *any* PR, right? Do you mind tweaking this, while at it?

Reply:
On second reading/in context, this is clear. Thanks!